presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -37,6 +37,7 @@
     private boolean binaryTransportEnabled;
     private DataSize maxTaskUpdateSize = new DataSize(16, MEGABYTE);
     private CommunicationProtocol taskCommunicationProtocol = CommunicationProtocol.HTTP;
+    private CommunicationProtocol serverInfoCommunicationProtocol = CommunicationProtocol.HTTP;
```

Comment:
Nit: I think it's reasonable to squash the commit that adds a config to the commit that first uses it.